radixdlt/babylon-wallet-android
# [ABW-3678] Reuse same view for dApp details screen and dApp details dialog

## Description
- reuse same UI for dapp details screen and dapp details dialog accessible from persona details.

Since this is `dialog` destination, it's not respecting edge to edge - I think we have similar issue with other dialogs that use `dialog`. This can't be fix easily without changing dialog to be part of the screen, unless I'm missing something here

## How to test

1. Verify that details opened by " personas -> persona detail -> dApp click " are the same as the one opened in from "Approved Dapps" screen. Verify that forget this dapp removes dApp from the persona and dismisses dialog.
